nifi-extension-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongoRecord.java

```diff
@@ -159,10 +152,7 @@ public class PutMongoRecord extends AbstractMongoProcessor {
         _propertyDescriptors.add(UPDATE_MODE);
         propertyDescriptors = Collections.unmodifiableList(_propertyDescriptors);
 
-        final Set<Relationship> _relationships = new HashSet<>();
-        _relationships.add(REL_SUCCESS);
-        _relationships.add(REL_FAILURE);
-        relationships = Collections.unmodifiableSet(_relationships);
+      relationships = Set.of(REL_SUCCESS, REL_FAILURE);
```

Comment:
This looks like it needs two extra spaces for indentation.
```suggestion
        relationships = Set.of(REL_SUCCESS, REL_FAILURE);
```